src/kgcl_schema/grammar/parser.py

```diff
@@ -54,6 +58,15 @@ def parse_statement(input: str) -> Change:
 
     Return an instantiated dataclass object from model.kgcl_schema.
     """
+    regex = r'<http[s]?://(?:[a-zA-Z]|[0-9]|[$-_@.&+]|[!*\(\),]|(?:%[0-9a-fA-F][0-9a-fA-F]))+'
+    uri = re.findall(regex, input)
```

Comment:
If the input string has multiple URI matches, only the first one will be dealt with

Reply:
I'm working with just adding a synonym as of now. Would there be a command like:

```
create synonym 'ABCD3' for http://purl.obolibrary.org/obo/FBbt_00004751 http://purl.obolibrary.org/obo/FBbt_1234
```

Reply:
no but there can easily be a statement like:

create edge http://purl.obolibrary.org/obo/FBbt_00004751 to http://purl.obolibrary.org/obo/FBbt_1234

Reply:
Addressed